src/components/illustration/illustration-files.tsx

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
I deliberately decided to split this file from  the export `illustration-set.tsx` where I do the types & keys processing so we can easily update this file without worrying about actual code that does things.